contrib/golang/filters/http/source/config.cc

```diff
@@ -33,7 +33,10 @@ Http::FilterFactoryCb GolangFilterConfig::createFilterFactoryFromProtoTyped(
       proto_config, dso_lib, fmt::format("{}golang.", stats_prefix), context);
   config->newGoPluginConfig();
   return [config, dso_lib](Http::FilterChainFactoryCallbacks& callbacks) {
-    auto filter = std::make_shared<Filter>(config, dso_lib);
+    const std::string worker_name = callbacks.dispatcher().name();
```

Comment:
no need to copy the worker_name; name() returns a const ref and you can use a ref for your temp.

Reply:
fixed